src/templates/daily_report/html.handlebars

```diff
@@ -0,0 +1,517 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+
+    <!-- Facebook sharing information tags -->
+    <meta property="og:title" content="*|MC:SUBJECT|*" />
+
+    <title>*|MC:SUBJECT|*</title>
+    <style type="text/css">
+        /* Client-specific Styles */
```

Comment:
It seems there is a lot of css in here, it may be good to remove any css that we aren't using.